plugins/modules/clustering/consul/consul_session.py

```diff
@@ -237,11 +243,12 @@ def remove_session(module):
                          session_id, e))
 
 
-def get_consul_api(module):
+def get_consul_api(module, token=None):
```

Comment:
What is the `token=None` parameter is doing here? It's not used anywhere in the code, nor in the implementation of this function.

Reply:
indeed, i've respected the signature of the one used in plugins/modules/clustering/consul/consul_kv.py
But instead I've fixed the signature of consul_kv